src/_project-names.ts

```
@@ -0,0 +1,299 @@
+		return aliasCache;
+	}
+
+	aliasCache = new Map();
+
+	const aliasEnv = (process.env[PROJECT_ALIASES_ENV] ?? '').trim();
+	if (aliasEnv === '') {
+		return aliasCache;
+	}
+
+	// Parse comma-separated name=alias pairs
+	const pairs = aliasEnv.split(',').map(pair => pair.trim()).filter(pair => pair !== '');
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)